git-extra/notepad

```diff
@@ -15,7 +15,12 @@ die "Usage: $0 <file>"
 
 if test -f "$1"
 then
-	unix2dos.exe "$1"
+	commitencoding="$(git config i18n.commitencoding)" ||
+		commitencoding=""
```

Comment:
I think it's fine this way. The return code is documented. Returning empty string is not as far as I saw.